DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
-\addtohist{}{2018-07-16}{Initial version.}{Belm, Comoretto}
-}
+\author{ Eric Bellm }
 
-\setDocCurator{Eric Bellm}
-\setDocUpstreamLocation{\url{https://github.com/lsst-dm/\lsstDocType-\lsstDocNum}}
-\setDocUpstreamVersion{\vcsrevision}
+\input history_and_info.tex
 
 
 \setDocAbstract{
-This is the test report for \milestoneId{} (\milestoneName{}), an LSST DM level 1 milestone pertaining to the \product{}.
+This is the test plan and report for \milestoneId{} (\milestoneName{}), an LSST level 2 milestone pertaining to the \product{} Subsystem.
 }
 
+
 \maketitle
 
 \section{Introduction}
 \label{sect:intro}
 
+
 \subsection{Objectives}
 \label{sect:objectives}
 
-This test activity demonstrates the successful execution of the major components of the Alert Distribution system at the scale of LSST Operations.
+This test activity demonstrates the successful execution of the major
+components of the Alert Distribution system at the scale of LSST
+Operations.\\[2\baselineskip]It will demonstrate that:\\
 
-It will demonstrate that:
 \begin{itemize}
-\item The Alert Distribution payloads can be made available on systems managed by the LSST Data Facility
-\item The Alert Distribution payloads can be executed under the control of the relevant LDF services
-\item All required science data products can be passed through the components of the alert distribution service.
+\tightlist
+\item
+  The Alert Distribution payloads can be made available on systems
+  managed by the LSST Data Facility
+\item
+  The Alert Distribution payloads can be executed under the control of
+  the relevant LDF services
+\item
+  All required science data products can be passed through the
+  components of the alert distribution service.
 \end{itemize}
 
-Note that this test plan does not extend to testing integration of the Alert Distribution System with the Alert Generation Science Pipeline or the LSST Science Platform.
-We also do not test the Alert Database at this time as its functional requirements are not sufficiently specified to enable testing.
+Note that this test plan does not extend to testing integration of the
+Alert Distribution System with the Alert Generation Science Pipeline or
+the LSST Science Platform.\\
+We also do not test the Alert Database at this time as its functional
+requirements are not sufficiently specified to enable
+testing.\\[2\baselineskip]
 
-\subsection{Scope}
-\label{sect:scope}
+\subsection{Scope}\label{scope}
 
-The overall test plan for the LSST Data Management system is described in \citeds{LDM-503}.
+The overall test plan for the LSST Data Management system is described
+in \citeds{LDM-503}.\\
+This document specifically refers to the milestone LDM-503-5, which
+tests the LSST Alert Dis- tribution System.\\
+The overall LSST Alert Distribution System test specification is defined
+in \citeds{LDM-533}.\\[2\baselineskip]This plan is identified in JIRA as LVV-P1.
+The related test run is LVV-R3 that foresee the execu- tion of the
```

Comment:
This paragraph is defined in Objective, and shall be removed, since the TPR template will get from Jira this information. I think I added it (sorry for the bad English) when docsteady was not ready for this type of documents.

Reply:
Just to clarify the LVV-R3. This test campaign has been planned, executed and documented, when the jira plugin was using Test Run objects (LVV-RX) instead of Test Cycles (LVV-CX). An update of the plugin changed the definition and migrate the existing data. Now LVV-R3 is LVV-C3.